### PR TITLE
Update improved resource stats to fully match civ5

### PR DESCRIPTION
**Pull request by ravignir**

### CODE REVIEW

File: android/assets/jsons/Civ V - Gods & Kings/TileResources.json [hunk 1]
```diff
@@ -474,7 +457,7 @@
 		"food": 1,
 		"gold": 1,
 		"improvement": "Plantation",
-		"improvementStats": {"food": 1,"gold": 1}
+		"improvementStats": {"food": 1}
```
Comment (will-ca) [inline android/assets/jsons/Civ V - Gods & Kings/TileResources.json:460]: Wiki says this should be +1 Gold (and then +1 Food only with Calendar)?

https://civilization.fandom.com/wiki/Cocoa_(Civ5)

…Hm. Also shouldn't exist without BNW, apparently.

File: android/assets/jsons/Civ V - Gods & Kings/TileResources.json [hunk 2]
```diff
@@ -501,7 +483,7 @@
 		"gold": 1,
 		"food": 1,
 		"improvement": "Mine",
-		"improvementStats": {"food": 1,"production":1},
+		"improvementStats": {"food": 1},
```
Comment (will-ca) [inline android/assets/jsons/Civ V - Gods & Kings/TileResources.json:486]: Wikia says the old stats?

https://civilization.fandom.com/wiki/Salt_(Civ5)

Comment (ravignir) [reply android/assets/jsons/Civ V - Gods & Kings/TileResources.json:485]: +1 production is provided by mine.

File: android/assets/jsons/Civ V - Gods & Kings/TileResources.json (hunk 1 above)
Comment (ravignir) [reply android/assets/jsons/Civ V - Gods & Kings/TileResources.json]: It is commented out anyway.

**Comment by ravignir** [comment]
Anyway, I would not merge it until I post all the screenshots from civ5, just to be sure :D

**Comment by ravignir** [comment]
G&K:
az
BNW:
za

Now:
Plantations have base yield of 1 gold in unciv.
Fishing Boats have base yield of 1 food.
Mines has base yield of 1 production.
Farms have base yield of 1 food.
Camps, Pastures, Quarries and Oil wells have no base yields.

**Review by yairm210 — APPROVED** [review]

**Pull request merged**